config/locales/en.yml

```diff
@@ -776,3 +766,92 @@ en:
 
             Alternatively, if your trade still does not meet the rules of
             origin, [start again](%{start_link}).
+
+    cumulation_methods:
+      heading: |
+        %{cumulation_method} cumulation
+      title: The %{scheme_title} allows for %{cumulation_method} cumulation with %{countries}.
+      example_title: |
+        %{cumulation_method} cumulation - an example
+      bilateral:
+        description_html: |
+          <p>
+            Bilateral cumulation operates between 2 countries or trading blocs where a 
+            trade agreement contains a provision allowing them to cumulate origin.
+          </p>
+
+          <p>
+            Materials originating in either country or bloc in the trade agreement will 
+            be considered as materials covered by the rule of origin.
+          </p>
+        example_list_md: |
+          1. A company in country A would like to take advantage of the FTA with country B to receive 0% import duties on the import of goods which require 100% of the value to be originating.
+          2. The goods are comprised of component parts that are sourced from both country A and country B
+          3. As there is bilateral cumulation between the two countries and all component parts are sourced in either country A or country B, the goods can be considered as originating.
+          4. Even if all the parts came from country A and were just assembled in country B, the rules would be met.
+      extended:
+        description_html: |
+            <p>
+              Extended cumulation (also referred to as third-party or cross cumulation) allows for
+               cumulation between countries which are not linked by a trade agreement or are linked 
+               by a trade agreement with different rules of origin to influence the origin of goods 
+               in the trade agreement between the two primary countries or blocs.
+            </p>
+
+            <p>
+              You are allowed to use inputs from a third-party country, which is not a member of the
+               applicable FTA and consider them originating, provided that they meet the rules of 
+               origin under the relevant trade agreement.
+            </p>
+
+            <p>
+              Extended cumulation is the most flexible type of cumulation of originating inputs and is 
+              often limited to certain tariff headings, subheadings and codes or to certain types of 
+              products only. Please check the detailed agreement text.
+            </p>
+        example_list_md: |
+          1. In order to take advantage of a 0% import duty, a business in Country A would like to import a product
+          from Country B where product-specific rules dictate that a maximum of 50% non-originating materials
+          is permitted in the item.
+          1. Only 35% of the components of the items were manufactured in Country B.
```

Comment:
Might be worth fixing up the numbers - markdown doesn't care (it just converts to `ol > li`) but would be neater